### PR TITLE
Joy: LeetCode/989/Java 

### PR DESCRIPTION
**Pull request by he2joojo**
Joy: LeetCode/989/Java

### CODE REVIEW

File: leetcode/Joy/week3/Leet989.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package leetcode.Joy.week3;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Leet989 {
```
Comment (ghkdgus29) [inline leetcode/Joy/week3/Leet989.java:6]: cool

Comment (othertkfka) [reply leetcode/Joy/week3/Leet989.java:6]: so cool

**Comment by won4885** [comment]
빅인티저

File: leetcode/Joy/week3/Leet989.java (hunk 1 above)
Comment (yeonise) [reply leetcode/Joy/week3/Leet989.java:6]: cool....

**Review by yeonise — COMMENTED** [review]
저도 double로 풀었어요

**Pull request merged**